Clean/Table/Basic.lean

```diff
@@ -402,6 +434,22 @@ instance [Repr F] : Repr (TableOperation S F) where
     | .EveryRow c => "EveryRow " ++ reprStr c
     | .EveryRowExceptLast c => "EveryRowExceptLast " ++ reprStr c
 
+instance : ToJson (TableOperation S F) where
+  toJson op := match op with
+    | .Boundary i c => Json.mkObj [
```

Comment:
nit: the idiomatic way to define functions whose arguments are immediately matched on is this:
```suggestion
  toJson
    | .Boundary i c => Json.mkObj [
```